packages/bind-mount/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @prairielearn/bind-mount
 
+## 1.1.0
+
+### Patch Changes
+
+- XXX: Convert from a CJS module to an ESM module
+
```

Comment:
Instead of updating CHANGELOG.md directly, you can run `make changeset` and let the changesets module take care of it.

Reply:
Good to know, thanks! [d5a92ba](https://github.com/PrairieLearn/PrairieLearn/pull/11068/commits/d5a92ba7cc12517fe1a64c1e0b77c89e14a7df36)